examples/thirdweb/lib/makeswift/register-components.tsx

```diff
@@ -34,7 +34,17 @@ ReactRuntime.registerComponent(NFTDropCard, {
       label: 'Show description',
       defaultValue: true,
     }),
-    showClaimed: Checkbox({ label: 'Show claimed', defaultValue: true }),
+    // showClaimed: Checkbox({ label: 'Show claimed', defaultValue: true }),
```

Comment:
Did you mean to leave this line commented out or remove it altogether?

Reply:
I didn't notice it, yes remove that line.

Reply:
Do you mind taking care of that change?

Reply:
Not yet, I was busy, I'll take a look into this today.